scripts/nolints.txt

```diff
@@ -1211,11 +1211,11 @@ apply_nolint multiplicity.finite doc_blame
 -- ring_theory/ring_invo.lean
 apply_nolint ring_invo.id doc_blame
 
--- ring_theory/unique_factorization_domain.lean
+-- ring_theory/unique_factorization_monoid.lean
```

Comment:
Please revert the changes to `nolints.txt` (thanks for adding all the docstrings by the way!).